apps/src/code-studio/initApp/project.js

```diff
@@ -960,6 +963,38 @@ var projects = (module.exports = {
       });
     });
   },
+
+  /**
+   * Tests whether provided sample code is different from the current project code.
+   * This also normalizes the code so incidental differences in line endings or
+   * empty xml tags are not recognized as differences.
+   * @param {string} sampleCode the code to diff against the current project code
+   */
+  isCodeDifferent(sampleCode) {
```

Comment:
NIT: `isCodeDifferent` begs the question "from what"? with a name like that i would expect it to take two arguments and compare them. a simple fix could be `isCurrentCodeDifferent`. however you could also factor this out into `parseCurrentCode` and `isCodeEqual`, which both seem potentially reusable.

Reply:
Ohhhh again, thanks for the name recommendations. `isCurrentCodeDifferent` sounds great.